New failure in test_mypy

## Bug report

**Required Info:**

- Operating System:
  - All platforms
- Installation type:
  - source
- Version or commit hash:
  - 54e80d50934b3c50a7e657a1682a548700a7ddab (current HEAD of `master`)
- DDS implementation:
  - N/A
- Client library (if applicable):
  - N/A

#### Steps to reproduce issue

The last two days worth of nightlies exhibit the issue on all platforms in all configurations: e.g.
* https://ci.ros2.org/view/nightly/job/nightly_linux_release/1623/
* https://ci.ros2.org/view/nightly/job/nightly_win_deb/1699/

#### Additional information


Here's a relevant-seeming extract from the build log

```
4 errors
test\conftest.py:25:2: error: No overload variant of "fixture" matches argument type "str"
test\conftest.py:25:2: note: Possible overload variants:
test\conftest.py:25:2: note:     def [_FixtureFunction] fixture(fixture_function: _FixtureFunction, *, scope: Union[Union[Literal['session'], Literal['package'], Literal['module'], Literal['class'], Literal['function']], Callable[[str, Config], Union[Literal['session'], Literal['package'], Literal['module'], Literal['class'], Literal['function']]]] = ..., params: Optional[Iterable[object]] = ..., autouse: bool = ..., ids: Union[Iterable[Union[None, str, float, int, bool]], Callable[[Any], Optional[object]], None] = ..., name: Optional[str] = ...) -> _FixtureFunction
test\conftest.py:25:2: note:     def fixture(fixture_function: None = ..., *, scope: Union[Union[Literal['session'], Literal['package'], Literal['module'], Literal['class'], Literal['function']], Callable[[str, Config], Union[Literal['session'], Literal['package'], Literal['module'], Literal['class'], Literal['function']]]] = ..., params: Optional[Iterable[object]] = ..., autouse: bool = ..., ids: Union[Iterable[Union[None, str, float, int, bool]], Callable[[Any], Optional[object]], None] = ..., name: Optional[str] = ...) -> FixtureFunctionMarker
```

--- a/sros2/test/conftest.py
+++ b/sros2/test/conftest.py
@@ -22,6 +22,6 @@ import pytest
 os.environ['HTTP_PROXY'] = 'http://example.com'
 
 
-@pytest.fixture('session')
+@pytest.fixture(scope='session')
 def test_policy_dir():
     return pathlib.Path(__file__).parent / 'policies'

